src/components/NavBar.js

```diff
@@ -58,9 +65,10 @@ const MenuItem = styled.ul`
      ) : (
        <Link
          to={to}
-         activeStyle={{
-           color: colors.darkTheme.primary
-         }}
+         style={{color: colors.tech47white}}
```

Comment:
@aayushineeshu I think we needed activeStyle also, as per this docs https://www.gatsbyjs.org/docs/gatsby-link/